addons/website_sale_slides/views/website_slides_templates.xml

```diff
@@ -125,6 +125,21 @@
                 <span t-field="channel.product_id.list_price" t-options="{'widget': 'monetary', 'display_currency': product_info['currency_id']}"/>
             </div>
         </div>
+        <t t-if="channel.prerequisite_channel_ids and not channel.has_completed_prerequisites">
+            <small t-if="len(channel.prerequisite_channel_ids) == 1" class="text-center">
+                Prerequisite:
+                <a t-attf-href="/slides/{{slug(channel.prerequisite_channel_ids[0])}}">
+                    <span t-out="channel.prerequisite_channel_ids[0].name"/>
```

Comment:
I think it can be simplify, because you already check above the length (same line just above)
```suggestion
                    <span t-out="channel.prerequisite_channel_ids.name"/>
```